### ISSUE TITLE
Stream relay configuration not work with oscam

### ISSUE THREAD

**Issue by DimitarCC** [OpenPLi/enigma2#3797]
https://github.com/OpenPLi/enigma2/blob/26f432591ce0fe383b20fabfe95c2d08e1586a01/lib/python/Screens/InfoBarGenerics.py#L161

This will not going to work with oscam for example. Since the service reference is not compatible.
The 7th column from the service reference string (1:0:19:83:6:85:**C0000**:0:0:0) have to be changed depending on the oscam requirements  so to work.

So for now the stream relay solution is not usable with oscam.

**Comment by littlesat** [comment]
IkNot fully true as here it works as a charm. It seems the binary you are using needs a fix. Changing this service ref will also arrange that the idea partly works. Eg on the 2md infobar you do not see transponder parameters as you see the relay url here. There is also no reason why this should be changed for the relay server.
It can be a requirement, but it is a requirement for a 'bug' in the oscam binary. I'm afraid it does double decryption once in the hardware and the second in the relay server and then indeed you need this work-a-round. So a fix in the emu is likely mandatory.
With the binary I'm using it works. 
My special user bouquets had also not had that changed.

**Comment by DimitarCC** [comment]
OK. Can you share what binary you using? 
With 11725 icam binary definitely not works....

**Comment by DimitarCC** [comment]
Also there is big issues in my opinion after you start that stream relayed channel..... It marks other channels as not playable and if you try to switch to them a "No free tuner" message appears.
Is there a workaround to that?

Maybe that is connected again with the emu handling...Thats why if you can share what emu binary you using or which version at least it will be great.

**Comment by littlesat** [comment]
That was always the case with streamrelay. But the message will disappear and picture will be shown

**Comment by DimitarCC** [comment]
No... it does not disappear all the time... Sometimes the destination service does not ever play.
Maybe is the matter of timeout...

**Comment by DimitarCC** [comment]
The problem with impossibility to switch to channel and permanent "No free tuner" message is because there is a need of time tuner to be freed usually, but i think that is implemented wrong since you should not try to zap at all and just delay it with about 100ms. In this case no "No free tuner" message will ever show up.

**Comment by littlesat** [comment]
You are suggesting here a work-a-round not  a solution….Verstuurd vanaf mijn iPhoneOp 18 okt. 2023 om 11:01 heeft Dimitar Tsenev ***@***.***> het volgende geschreven:﻿
The problem with impossibility to switch to channel and permanent "No free tuner" message is because there is a need of time tuner to be freed usually, but i think that is implemented wrong since you should not try to zap at all and just delay it with about 100ms. In this case no "No free tuner" message will ever show up.

**Comment by DimitarCC** [comment]
It is work around at the moment too... At least in the part of that delayed zap....
